node/node.go

```diff
@@ -828,6 +845,12 @@ func (node *Node) setupForBeaconValidator() {
 		utils.GetLogInstance().Error("create group receiver error", "msg", err)
 		return
 	}
+	node.clientReceiver, err = node.host.GroupReceiver(p2p.GroupIDBeaconClient)
```

Comment:
Duplicated code. This and the above can be combined into a method.

Reply:
sure.